crates/sources/src/inline.rs

```diff
@@ -36,22 +45,44 @@ pub fn inline_capture(
 
     match endpoint {
         models::CaptureEndpoint::Connector(models::ConnectorConfig { config, .. }) => {
-            inline_config(config, scope, resources)
-        }
-        models::CaptureEndpoint::Local(models::LocalConfig { config, .. }) => {
-            inline_config(config, scope, resources)
+            inline_config(
+                Scope::new(scope)
```

Comment:
yep, it's because the inner `scope` is a `&url::Url`, while the `Scope` type wraps it with location atoms that defer actually building a Url until it's flattened.